chrome/script.browserify.js

```diff
@@ -86,15 +96,77 @@ function view() {
   ]);
 }
 
+function filterLogins(e) {
+  // remove executed search from input field
+  if (!fillOnSubmit && e.target.value.indexOf(domain) === 0) {
+    e.target.value = e.target.value.substr(domain.length);
+  }
+
+  // use fuzzy search to filter results
+  var filter = e.target.value.trim().split(/[\s\/]+/);
+  if (filter.length > 0) {
+    logins = resultLogins.slice(0);
+    filter.forEach(function(word) {
+      if (word.length > 0) {
+        var refine = [];
+        FuzzySort.go(word, logins, {allowTypo: false}).forEach(function(result) {
+          refine.push(result.target);
+        });
+        logins = refine.slice(0);
+      }
+    });
+
+    // fill login forms on submit rather than initiating a search
+    fillOnSubmit = logins.length > 0;
+  } else {
+    // reset the result list if the filter is empty
+    logins = resultLogins.slice(0);
+  }
+
+  // redraw the list
+  m.redraw();
+
+  // show / hide the filter hint
+  showFilterHint(logins && logins.length);
+}
+
+function searchKeyHandler(e) {
+  // switch to search mode if '\' is pressed and no filter text has been entered
+  if (e.code == "Backspace" && logins && logins.length > 0 && (!e.target.value.length || e.target.value == domain)) {
```

Comment:
- [ ] I think `&& logins` is not needed anymore since it can never be null
- [ ] `switch to search mode if '\' is pressed` - needs to be `Backspace`

Reply:
Done.